snappy/security.go

```diff
@@ -319,6 +320,13 @@ func findWhitespacePrefix(t string, s string) string {
 	return subs[1]
 }
 
+// XXX: what to use really?
+func getSecurityProfile2(s *snap.Info, appName, baseDir string) string {
```

Comment:
@zyga no, I have the reverse problem, all the functions generating wrappers use this to feed the value into the wrappers themselves, what should I use nowadays? or should we have a method on Info for it instead?